react/react1/week1/static-todo-list/src/App.js

```diff
@@ -0,0 +1,8 @@
+// import './App.css';
+import ToDoWrapper from './components/ToDoWrapper';
+
```

Comment:
I know it's not necessary but I still write `import React from 'react';` line in every component, just for the sake of consistency :)